linkerd/core/src/main/scala/io/buoyant/linkerd/IdentifierInitializer.scala

```diff
@@ -0,0 +1,5 @@
+package io.buoyant.linkerd
+
+import io.buoyant.linkerd.config.ConfigInitializer
+
+trait IdentifierInitializer extends ConfigInitializer
```

Comment:
Found it in Intellij:
Settings→Editor→General→Other→Ensure line feed at file end on save.